.github/PULL_REQUEST_TEMPLATE/pull-request.md

```diff
@@ -0,0 +1,48 @@
+<!-- Provide a general summary of the changes in the Title above. -->
+
+## Related Issue(s)
+
+<!-- Please link to the issue(s) where this feature/bug is discussed/reported. -->
+
+Resolves #1
+
+## Description
+
+<!--
+If not clearly described in the related issue(s), describe:
+- Why this change is propsed?
+- What issue/bug does it solve?
+-->
+
+## Screenshots/Screencasts
+
+<!-- Copy/paste or drag/drop images/videos here to assist in reviewing this pull request. -->
+
+## Types of changes
+
+<!--
+What types of changes does your code introduce?
+Put an `x` in all the boxes that apply.
+-->
+
+- [ ] Bug fix.
+- [ ] New feature or updated functionality.
+
+**Is this a breaking change?** eg. Will this change require updates to existing
+election configuration files?
+
+- [ ] Yes, this is a breaking change.
+
+<!-- If so, describe what will break and steps to fix. -->
+
+## Checklist:
+
+<!-- Put an `x` in all the boxes that apply. -->
+<!-- If unsure about any, don't hesitate to ask. We're here to help! -->
+
```

Comment:
this feels like a high bar for a PR, might lead to folks just not submitting them. Are we sure we need this level of detail?

Reply:
Yeah, I feel that too. I've added an additional comment to the top to add clarification. If that suffices, please LGTM. If not, please suggest otherwise or push a commit with your suggestion.

Reply:
I can live with it, let's see how we do!